tornado_swirl/docparser.py

```diff
@@ -60,7 +60,7 @@ def __new__(cls, val):
 
 _PROPS_TYPE_LOOKUP = {
     Boolean: ('exclusiveMinimum', 'exclusiveMaximum', 'uniqueItems',
-              'allowEmptyValue', 'deprecated', 'readOnly', 'writeOnly'),
+              'allowEmptyValue', 'deprecated', 'readOnly', 'writeOnly', 'additionalProperties'),
```

Comment:
Why is Boolean relevant here?

Reply:
because we're setting the attribute as a boolean type. this is so that it converts the string to boolean. Technically it can be something other than boolean but for our purposes it's likely only to be boolean.